manifests/config.pp

```diff
@@ -30,4 +30,6 @@
     mode   => '0755',
   }
 
+  $services_real = prefix($katello::services_list, $katello::services_directory)
+  ensure_resource('file', $services_real, {'ensure' => 'file', 'mode' => '0644'})
```

Comment:
It could be that puppet-katello and puppet-capsule end up with overlapping services.